crypto/asn1/asn_mime.c

```diff
@@ -613,9 +613,9 @@ static int multi_split(BIO *bio, const char *bound, STACK_OF(BIO) **ret)
                 return 0;
             }
             return 1;
-        } else if (part) {
-            /* Strip CR+LF from linebuf */
-            next_eol = strip_eol(linebuf, &len, 0);
+        } else if (part != 0) {
+            /* Strip (possibly CR +) LF from linebuf */
+                next_eol = strip_eol(linebuf, &len, flags);
```

Comment:
indent incorrect..

Reply:
Corrected.
On this occasion I also corrected a couple of pre-existing coding style issues in `asn_mine.c`.